lineup.js

```diff
@@ -1,9 +1,25 @@
 function calculateTotalSalary(lineup) {
-  return 0
+  return lineup.reduce((total, player) => {
+    return total + player.salary
+  }, 0)
 }
 
 function getPositionCounts(lineup) {
-  return {}
+  return lineup.reduce((positionCounts, player) => {
+
+    positionCounts[player.position] = positionCounts[player.position] === undefined ? 1 : positionCounts[player.position] + 1
+    return positionCounts
+
+
+    /*
```

Comment:
Always delete commented out code before creating a PR. While it can be helpful during the writing process, we wouldn't want it merged and left hanging around.